tests/hana_test.py

```diff
@@ -505,7 +507,46 @@ def __init__(self, rc):
             self._hana._run_hana_command.assert_called_once_with(
                 'HDBSettings.sh systemReplicationStatus.py', exception=False)
             self.assertEqual(status, {"status": expect})
+    
+    def test_set_ini_parameter(self):
```

Comment:
I would like to see other test scenarios where `reconfig` and `layer` options are different, to cover all of the use case.

Reply:
I added the scenarios where reconfig and layer/layer_name params vary, for both set and unset methods